pkg/oc/cli/admin/release/extract.go

```diff
@@ -147,3 +206,171 @@ func (o *ExtractOptions) Run() error {
 		return opts.Run()
 	}
 }
+
+func (o *ExtractOptions) extractGit(dir string) error {
+	if err := os.MkdirAll(dir, 0750); err != nil {
+		return err
+	}
+
+	release, err := NewInfoOptions(o.IOStreams).LoadReleaseInfo(o.From)
+	if err != nil {
+		return err
+	}
+
+	cloner := &git{}
+
+	hadErrors := false
+	alreadyExtracted := make(map[string]string)
+	for _, ref := range release.References.Spec.Tags {
+		repo := ref.Annotations["io.openshift.build.source-location"]
+		commit := ref.Annotations["io.openshift.build.commit.id"]
+		if len(repo) == 0 || len(commit) == 0 {
+			if glog.V(2) {
+				glog.Infof("Tag %s has no source info", ref.Name)
```

Comment:
nit: i would print the `ref.Annotations` here

Reply:
Why?

Reply:
because maybe there is a typo but this won’t  tell me where